cmd/options.go

```diff
@@ -82,3 +82,10 @@ func WithQuietLogging() Option {
 		c.conf.Quiet = true
 	}
 }
+
+// WithDebugLogging configures the Proxy to log debug level messages.
+func WithDebugLogging() Option {
```

Comment:
went with name `WithDebugLogging` for the option over `WithDebugLogs` since we already have an option called `WithQuietLogging` and thought we should match naming.